.travis.yml

```diff
@@ -2,12 +2,16 @@ language: android
 android:
   components:
     - build-tools-22.0.1
-    - android-19
-    - android-16
+    # old build tools due to core lib
+    - build-tools-20.0.0
+    - android-22
+    # Additional components
+    - extra-google-m2repository
```

Comment:
For the appcompat dependiencies it made sense to me to explicitly reference the google repos.